website/content/docs/concepts/mount-migration.mdx

```diff
@@ -0,0 +1,86 @@
+---
+layout: docs
+page_title: Mount Migration
+description: Move secrets and auth mounts within and across namespaces
+---
+
+# Mount Migration
+
+Vault supports moving secrets and auth mounts, both within and across namespaces. The operation
+is supported by API endpoints as well as a CLI helper around them.
+
+## Why
+
+There can be several reasons for wanting to migrate mounts. On OSS, the use cases could be for 
+renaming mounts in order to align with org standards. 
+On Enterprise, there are a few more potential use cases as namespaces come into the picture. 
+When migrating from an OSS binary to an Enterprise binary, organizations may want to divide their 
+mounts across several namespaces. Mounts may also be moved across namespaces when there is a reorganization
+of roles and responsibilities.
+
+## Async Behavior
+
+The first thing to note about the `sys/remount` endpoint is that it is an asynchronous endpoint. An invocation
+will start the migration process, and the API will return a migration ID. That ID can in turn be used to monitor
+the status of the migration using the `sys/remount/status` endpoint.
+
+## Namespaces
+
+The `sys/remount` endpoint can move mounts across namespaces with the namespaces supplied as prefixes
+on the path arguments. In this case, namespaces for both the source and destination mounts need to be 
+either the namespace in which the `/sys/remount` endpoint is called, or the children of that namespace. Let us assume that 
+we have 3 namespaces ns1/, ns1/ns2/ and ns1/ns3. We want to move the mount `my-secret` from ns1/ns2/my-secret to ns1/ns3/my-secret. 
+This can be done in one of two ways. The first way is to invoke `/sys/remount` from the root namespace with the `from` argument as ns1/ns2/my-secret 
+and `to` argument as ns1/ns3/my-secret. The second way is to invoke `/sys/remount` from the namespace ns1/, with `from` as ns2/my-secret and `to` 
+as ns3/my-secret.
+
+## Leases
+
+The remount operation will result in existing leases on a mount being revoked. This is potentially the 
```

Comment:
```suggestion
The remount operation will result in existing leases for a mount being revoked. Please note that remounting may be a consuming process. 
```

Reply:
I think this should stay as is, or be reworded in a different way? I do want it clear that if the remount is taking long it may be cause of this. How about something like this ?
```
On mounts with a large number of leases, the revocation process and hence remounting as a whole may be a time consuming process
```

Reply:
```suggestion
The remount operation will result in existing leases on a mount being revoked. On mounts with a large number of leases, the revocation process and hence remounting as a whole may be a time consuming process.
```